src/main/kotlin/defineModelWithDeprecatedSyntax.kt

```diff
@@ -44,10 +43,8 @@ fun defineModelWithDeprecatedSyntax(model: Model) {
   courtAdmin.uses(Delius.system, "records CAS decision, referrals in")
   courtAdmin.uses(prepareCaseForCourt, "captures court judgements in")
   ProbationPractitioners.crc.uses(oasys, "records offender risk (attendance, contact, etc.) and assessment in")
-  crcProgrammeManager.interactsWith(deliusSupport, "opens tickets to update interventions")
+  crcProgrammeManager.interactsWith(Delius.supportTeam, "opens tickets to update interventions")
```

Comment:
This whole line, particularly `Delius.supportTeam`, reads really clearly  👍

Reply:
Thank you :)